Windows compilation error

Hi Wenzel,

When compiling the examples with VS2015, I have the following error:

pybind11/numpy.h(152): error C2133: ' ?? ?? :: ?? ::values': unknown size

due to the following declaration:

```cpp
    constexpr static const int values[] = {
        array::API::NPY_BYTE_, array::API::NPY_UBYTE_, array::API::NPY_SHORT_,    array::API::NPY_USHORT_,
        array::API::NPY_INT_,  array::API::NPY_UINT_,  array::API::NPY_LONGLONG_, array::API::NPY_ULONGLONG_ };
```

--- a/include/pybind11/numpy.h
+++ b/include/pybind11/numpy.h
@@ -147,7 +147,7 @@ NAMESPACE_BEGIN(detail)
 
 template <typename T> struct npy_format_descriptor<T, typename std::enable_if<std::is_integral<T>::value>::type> {
 private:
-    constexpr static const int values[] = {
+    constexpr static const int values[8] = {
         array::API::NPY_BYTE_, array::API::NPY_UBYTE_, array::API::NPY_SHORT_,    array::API::NPY_USHORT_,
         array::API::NPY_INT_,  array::API::NPY_UINT_,  array::API::NPY_LONGLONG_, array::API::NPY_ULONGLONG_ };
 public:

